server/graphql/types/src/types/program/program_enrolment.rs

```diff
@@ -202,6 +214,22 @@ impl ProgramEnrolmentNode {
         &self.program_enrolment.0.patient_id
     }
 
+    pub async fn patient(&self, ctx: &Context<'_>) -> Result<PatientNode> {
+        let loader = ctx.get_loader::<DataLoader<PatientLoader>>();
+
+        let result = loader
+            .load_one(self.program_enrolment.0.patient_id.clone())
+            .await?
+            .map(|patient| PatientNode {
+                store_id: self.store_id.clone(),
```

Comment:
Its because the patient node has nodes that require the current store id. As far as I can see its the clinician node to load the clinicians for the current store...